R/split_var_call.R

```diff
@@ -0,0 +1,60 @@
+R_SPLIT = function(domain_dataset,max_length_out = 200){

```

Comment:
- Please use tidyver styleguid and snake case https://style.tidyverse.org/syntax.html
- Please add roxygen documentation, with examples
- Please add tests cases
- Please add namesacpes:: to all functions calls from tidyverse packages